src/styles/index.scss

```diff
@@ -1,3 +1,114 @@
-body {
+@import './variables';
+@import './mixin';
+
+// style
+* {
   margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: $body-font;
+}
+
+.product {
```

Comment:
it would be best if you moved each bem block to separate file

Reply:
done

Reply:
done